apps/site/assets/css/_schedule-page.scss

```diff
@@ -563,3 +563,7 @@
     }
   }
 }
+
+.pull-left {
```

Comment:
nitpick: If this isn't going to be nested or use a BEM name I think this might want to go in a CSS file outside of schedule specific css.
Put it in a file like `utilities.scss`, or nest it under the `schedule-table__cell` class
I say this because I already see we have a class called `pull-left` nested in the `datepicker.scss` file